Source/Core/VideoBackends/Software/SWmain.cpp

```diff
@@ -41,7 +41,7 @@
 
 #define VSYNC_ENABLED 0
 
-static volatile u32 s_swapRequested = false;
+static std::atomic<bool> s_swapRequested;
 
 static volatile struct
```

Comment:
At least the videocommon (former) one did require to be atomic: This flag is raised when the VI (on the cpu) wants to swap the framebuffer. As this must be done on the GPU, it often poll for this flag.